### ISSUE TITLE
TestTimeAugmentation assumes label is present

### ISSUE THREAD

**Issue by rijobro** [Project-MONAI/MONAI#1979]
**Describe the bug**
During training/testing/validation, ground truth labels are often present. However, at deployment only the image will be present. The way TTA is currently set up assumes that both image and label are present, which needs to be modified.

cc @diazandr3s

**Comment by wyli** [comment]
do you think this could be addressed by `InvertD` @Nic-Ma ?  https://github.com/Project-MONAI/MONAI/blob/a566dc69d19c7437a9b6b7dcd5dbfd7a9f077bab/monai/transforms/post/dictionary.py#L392

**Comment by Nic-Ma** [comment]
Hi @wyli ,

You are right, let me try to update the `Invertd` ASAP, and I also double confirm the TTA logic in the mean time.

Thanks.

**Comment by Nic-Ma** [comment]
Hi @rijobro ,

I checked the TTA code and this part confused me:
```py
# do model forward pass
batch_output = self.inferrer_fn(batch_images)
if isinstance(batch_output, torch.Tensor):
    batch_output = batch_output.detach().cpu()
if isinstance(batch_output, np.ndarray):
    batch_output = torch.Tensor(batch_output)
```
If the output is Tensor, it will be changed to numpy array and then change back to Tensor?

Thanks.
